lightkurve/targetpixelfile.py

```diff
@@ -1130,6 +1131,163 @@ def cutout(self, center=None, size=5):
         return self.__class__(newfits)
 
 
+    def plot_pixels(self, ax=None, normalize=False, periodogram=False, aperture_mask=None, show_flux=False, corrector_func=None, style='lightkurve', title=None, **kwargs):
+        """ Plot the light curves or associated periodograms for each pixel in one quarter
+        Note that all values are autoscaled and axis labels are not provided.
+        This utility is designed for by-eye inspection of signal morphology.
+        
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            A matplotlib axes object to plot into. If no axes is provided,
+            a new one will be generated.
+        normalized : bool
+            Default: False; if True, the normalized light curves will be plotted.
+        periodogram : bool
+            Default: False; if True, periodograms will be plotted, using normalized light curves.
+            Note that this keyword overrides normalized.
+        aperture_mask : ndarray or str
+            Highlight pixels selected by aperture_mask.
+            Only `pipeline`, `threshold`, or custom masks will be plotted.
+            `all` and None masks will be ignored.
+        show_flux : bool
+            Default: False; if True, shade pixels with frame 0 flux colour
+            Inspired by https://github.com/noraeisner/LATTE
+        corrector_func : function
+            Function that accepts and returns a `~lightkurve.lightcurve.LightCurve`.
+            This function is applied to each light curve in the collection
+            prior to stitching. The default is to normalize each light curve.            
+        style : str
+            Path or URL to a matplotlib style file, or name of one of
+            matplotlib's built-in stylesheets (e.g. 'ggplot').
+            Lightkurve's custom stylesheet is used by default.
+        kwargs : dict
+            e.g. `threshold` to be passed to create_threshold_mask
+        
+        """
+
+        if style == 'lightkurve' or style is None:
+            style = MPLSTYLE 
+        
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+
+            # get an aperture mask for each pixel
+            masks = np.zeros((self.shape[1]*self.shape[2], self.shape[1], self.shape[2]), dtype='bool')
+            for i in range(self.shape[1]*self.shape[2]):
+                masks[i][np.unravel_index(i, (self.shape[1], self.shape[2]))] = True
+            
+            pixel_list = []
+
+            for j in range(self.shape[1]*self.shape[2]):
+
+                lc = self.to_lightcurve(aperture_mask=masks[j])
+
+                if normalize: # overrides corrector function
```

Comment:
Let's change this into:
```suggestion
                if normalize or corrector_func is None: # overrides corrector function
```
...which would allow us to get rid of lines 1189-1191, which reduces code duplication! :+1: